sites/kit.svelte.dev/scripts/types/index.js

```diff
@@ -55,6 +55,10 @@ async function get_types(code, statements) {
 				// @ts-ignore no idea why it's complaining here
 				const name = name_node.name?.escapedText;
 
+				if (name === 'error') {
+					console.log(statement);
+				}
+
```

Comment:
Looks like a leftover from debugging
```suggestion
```

Reply:
Thanks! I always forget to remove these.